SmartDeviceLink/SDLStreamingVideoLifecycleManager.m

```diff
@@ -704,6 +704,8 @@ - (void)sdl_stopVideoSession {
 
     if (self.isVideoConnected || self.isVideoSuspended) {
         [self.videoStreamStateMachine transitionToState:SDLVideoStreamManagerStateShuttingDown];
+    } else if ([self.videoStreamStateMachine.currentState isEqualToEnum: SDLVideoStreamManagerStateStarting]) {
```

Comment:
```suggestion
    } else if ([self.videoStreamStateMachine.currentState isEqualToEnum:SDLVideoStreamManagerStateStarting]) {
```

Reply:
This will create a race condition. When the manager is in the starting state, there's an async call to get a video capabilities. If this happens while waiting for that callback to occur, the state will change from starting -> stopped -> then will send a start service message. I would recommend adding a check at the start of that callback to see if it's still in the starting state. If it's not, then it would abort.